app/gui/src/project-view/components/CodeEditor/CodeEditorImpl.vue

```diff
@@ -57,7 +60,7 @@ onMounted(() => {
 </script>
 
 <template>
-  <CodeMirror ref="editorRoot" class="CodeEditor" />
+  <CodeMirror ref="editorRoot" class="CodeEditor" @keydown.tab.stop.prevent />
```

Comment:
Why only tab is stop-prevented here? Some time ago, we needed to stop also `enter`, `backspace`, `delete` and arrows, because they interfered with adding/deleting/moving nodes.

Reply:
Those events are stopped in the `CodeMirror` component, `tab` is different because we don't want all `CodeMirror` editors to handle it.

Reply:
Ah, I see. I forgot `CodeMirror` is actually _our_ component...